### ISSUE TITLE
[Question] How can I first point the locator to the frame inside the frame before starting codegen?

### ISSUE THREAD

**Issue by bbchuoo** [microsoft/playwright#33241]
I have three HTML files: content.html, main.html, and test.html.

The homepage is test.html and its content is
```
<html>
<head>
   <title>HTML Frames</title>
</head>
<frameset cols="25%,50%,25%">
   <frame name="main" src="./main.html" />
</frameset>
</html>
```

main.html its content is
```
<frameset cols="25%,50%,25%">
    <frame name="content" src="./content.html" />
 </frameset>
```

content.html its content is
```
name：<input>
year：<input>
```

I want to point the locator to the innermost content frame and use await page.pause(); to start codegen. I want to use codegen to get the fields name or year from content.html without writing .getByText() myself. 

I tried the following code. Although it opens codegen, the locator remains in the main frame and cannot capture the fields in the content frame.

```
const browser = await chromium.launch({ headless: false });
const context = await browser.newContext();
const newPage = await context.newPage();
await newPage.frameLocator('frame[name="main"]').frameLocator('frame[name="content"]');
await newPage.pause();
```

How should I do this? Thank you 🙂

**Comment by mxschmitt** [comment]
What do you mean with "point the locator to the innermost content frame"? Codegen can only operate on a page object, you cannot start it on a specific frame.

This line won't do anything:

    await newPage.frameLocator('frame[name="main"]').frameLocator('frame[name="content"]');

When create the frames as per above and interact with the inputs, it generates the right locators for me, with the frames inside. do you mind elaborating what you are trying to do / what is not working with expected / actual outcomes?

**Comment by Skn0tt** [comment]
Closing as stale, please open a feature request if you think something's missing in Playwright, and include lots of info on your usecase.
